common/djangoapps/student/models.py

```diff
@@ -1503,6 +1503,10 @@ def refund_cutoff_date(self):
             attribute = self.attributes.get(namespace='order', name='order_number')
         except ObjectDoesNotExist:
             return None
+        except MultipleObjectsReturned:
+            # If there are multiple attributes then return the last one.
+            log.exception("MultipleObjectsReturned Exception")
```

Comment:
I think "warning" makes more sense, with a log message that explains what's happening and why we're throwing it.